### PR TITLE
EN6-51 edit content model

### PR DESCRIPTION
**Pull request by jeffgo10**
FYI, #8 needs to be approved & merged before going through this PR. This is the editing feature for content models

### CODE REVIEW

File: jsconfig.json [hunk 1]
```diff
@@ -2,5 +2,5 @@
   "compilerOptions": {
     "baseUrl": "src"
   },
-  "include": ["src", "test"]
+  "include": ["src"]
```
Comment (nicpuddu) [inline jsconfig.json]: why are we making this change?

Comment (jeffgo10) [reply jsconfig.json]: we don't actually have a `test` folder. So I removed it. This is just for vscode, by the way. https://code.visualstudio.com/docs/languages/jsconfig

File: src/state/content-model/types.js [hunk 2]
```diff
@@ -1,2 +1,3 @@
 // eslint-disable-next-line import/prefer-default-export
```
Comment (nicpuddu) [inline src/state/content-model/types.js]: this comment is no longer necessary since there are two exports

Comment (jeffgo10) [reply src/state/content-model/types.js]: done

**Pull request merged**